src/System.Memory/src/System/MemoryDebugView.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System
+{
+    internal sealed class MemoryDebugView<T>
+    {
+        private readonly ReadOnlyMemory<T> _memory;
+
+        public MemoryDebugView(Memory<T> memory)
+        {
+            _memory = memory;
+        }
+
+        public MemoryDebugView(ReadOnlyMemory<T> memory)
+        {
+            _memory = memory;
+        }
+
+        [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
+        public T[] Items
+        {
+            // This is a work around since we cannot use _memory.ToArray() due to VSTS bug 286592
```

Comment:
Nit: There are many different VSTS instances - the comment should say which one is it. E.g.:

`https://devdiv.visualstudio.com/DevDiv/_workitems?id=286592`

(I am aware of that not everybody has access to this URL, but I think it is better than nothing and there are existing places where we are using similar URLs.)